GitExtUtils/GitUI/Theming/ThemeFix.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Windows.Forms;
+using GitUI;
+
+namespace GitExtUtils.GitUI.Theming
+{
+    public static class ThemeFix
+    {
+        private static readonly ConditionalWeakTable<IWin32Window, IWin32Window> Ready =
```

Comment:
`Ready` is a really vague name.
Perhaps `ThemedControls` or `ProcessedControls`?

Reply:
Indeed it is vague. Renamed to `AlreadyFixedControls`